tests/cephadm/test_cephadm_pull.py

```diff
@@ -0,0 +1,59 @@
+import requests
+import yaml
+
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class CephadmpullFailure(Exception):
+    pass
+
+
+def _get_latest_ceph_version(url):
+    """
+    Get latest ceph version from URL
+    Args:
+        url: Ceph container yaml link
+    Returns:
+        Latest ceph version
+    """
+
+    response = requests.get(url)
+    yaml_response = yaml.safe_load(response.text)
+    log.info(yaml_response)
```

Comment:
Add some info about `yaml_response` in message

Reply:
Updated